### ISSUE TITLE
[Suggestion]: Add enchantment compatibility to the Platinum Infused Hatchet

### ISSUE THREAD

**Issue by CourierNatix** [Fuzss/illagerinvasion#46]
### Mod Loader (Optional)

Fabric

### Minecraft Version(s) (Optional)

1.20.1

### Mod Version(s) (Optional)

v8.0.4

### Suggestion (Required)

I have Illager Invasion mod on a server and I want to use the Platinum Infused Hatchet more, but the weapon isn't practical in any way. It's a great alternative to a trident in theory, with potential for **Loyalty and Axe enchants**. They're such a rare drop too making them fun to collect. Currently the only 2 useful enchants I can add are Mending and Unbreaking since they have durability. Besides throwing them at my target dummy or my friends for fun, I wanna use them for combat.

**Comment by zHawky** [comment]
Remember one [ancient but funny mod](https://github.com/Zot201/Tomahawks), maybe it would be interesting to implement it from here?

**Comment by Fuzss** [comment]
I've enabled support for axe weapon enchantments in v21.3.0, supporting some trident enchantments is a chore though, not sure about that. Maybe some unique enchantments in the future.

**Comment by NoBar94** [comment]
Bumping this as these were my thoughts exactly. Really what would be thematically cool is a "Returning" enchantment (same thing as loyalty, but for Axes) and maybe something like Breach as it would do heavy damage to armor? Love all your mods Fuzs!

**Comment by BigJMoney** [comment]
I disagree that it should get (at least all of the) trident enchantments, because then it's just a boring trident replacement. The hatchet should fill its own niche rather than be just "a trident with axe enchantments". The trident isn't worth copying anyway, as its main role is to kill sea creatures or as a poor man's Elytra.
